Source/objects.h

```diff
@@ -9,6 +9,8 @@
 
 namespace devilution {
 
+#define MAX_LVLS 24

```

Comment:
Isn't this level related? Why objects.h?

Reply:
it appears to only be used to define placment of shrines.

Reply:
Shouldn't it still be moved to a level-related header though? I think what matters most is the semantic of the value, not its use.

Reply:
We could rename it SHRINE_END_LEVEL if that would help.